maintenance/enabling-bpf.md

```diff
@@ -0,0 +1,274 @@
+
+### Before you begin...
+
+eBPF mode has the following pre-requisites:
+
+- A supported Linux distribution:
+  
+  - Ubuntu 20.04.
+  - Red Hat v8.2 with Linux kernel v4.18.0-193 or above (Red Hat have backported the required features to that build).
+  - Another [supported distribution]({{site.baseurl}}/getting-started/kubernetes/requirements) with Linux kernel v5.3 or above.
+    
+  If {{site.prodname}} does not detect a compatible kernel, {{site.prodname}} will emit a warning and fall back to standard linux networking.
+  
+- On each node, the BPF filesystem must be mounted at `/sys/fs/bpf`.  This is required so that the BPF filesystem persists when {{site.prodname}} is restarted.  (If the filesystem does not persist then pods will temporarily lose connectivity when {{site.prodname}} is restarted.)
+- For best pod-to-pod performance, an underlying network that doesn't require Calico to use an overlay.  For example:
+ 
+  - A cluster within a single AWS subnet.
+  - A cluster using a compatible cloud provider's CNI (such as the AWS VPC CNI plugin).
+  - An on-prem cluster with BGP peering configured.
+  
+- The underlying network must be configured to allow VXLAN packets between {{site.prodname}} hosts.  In eBPF mode, VXLAN is used to forward traffic to Kubernetes NodePorts, while preserving source IP.
+- A stable way to address the Kubernetes API server. Since eBPF mode takes over from kube-proxy, {{site.prodname}} needs a way to reach the API server directly.
+- The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
+
+> **Note**: The default kernel used by EKS is not compatible with eBPF mode.  If you wish to try eBPF mode with EKS, follow [these instructions](./ebpf-and-eks).
+{: .alert .alert-info}
+
+### How to
+
+- [Verify that your cluster is ready for eBPF mode](#verify-that-your-cluster-is-ready-for-ebpf-mode)
+- [Configure {{site.prodname}} to talk directly to the API server](#configure-{{site.prodnamedash}}-to-talk-directly-to-the-api-server)
+- [Configure kube-proxy](#configure-kube-proxy)
+- [Enable eBPF mode](#enable-ebpf-mode)
+- [Try out DSR mode](#try-out-dsr-mode)
+- [Reversing the process](#reversing-the-process)
+
+#### Verify that your cluster is ready for eBPF mode
+
+This section explains how to make sure your cluster is suitable for eBPF mode.
+
+1. To check that the kernel on a node is suitable, you can run 
+
+   ```bash
+   uname -rv
+   ```
+   
+   The output should look like this:
+   
+   ```
+   5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020
+   ```
+   
+   In this case the kernel version is v5.4, which is suitable.
```

Comment:
Do we specify the oldest Linux kernel version that support eBPF + Calico networking? If not, that would be useful. This addition can happen after merging this PR.

Reply:
Yes, it's in the Before you begin section